modules/configuration/php/NDB_Form_configuration.class.inc

```diff
@@ -106,6 +106,47 @@ class NDB_Form_Configuration extends NDB_Form
         $this->tpl_data['sandbox']         = $config->getSetting("sandbox");
 
     }
+
+    function project() {
+        $factory = NDB_Factory::singleton();
+        $config = $factory->config();
+	
+	/*if($config->getSetting(useProject)){*/
+	    $projectList = Utility::getprojectList();
```

Comment:
About the useProject. Is this function supposed to give all the projects of every studies or give all the projects of that study?